docs/source/troubleshooting/troubleshooting-misc.adoc

```diff
@@ -33,3 +33,31 @@ If you want to perform development tasks inside the {project} VM, it is recommen
 Depending on your operating system and shell environment, certain special characters can trigger variable interpolation and therefore cause passwords to fail.
 
 Workaround: When creating and entering passwords, wrap the string with single quotes in the following format: `'<password>'`
+
+
+[[authentication-required-to-push-image]]
+== Authentication required to push image to OpenShift integrated Docker registry
+
+If you are using Fedora, CentOS or RHEL host and you are running {project} with the Boot2Docker or Minikube xref:../using/basic-usage.adoc#choosing-iso-image#[ISO] 
+then even after following the steps defined in xref:../openshift/openshift-docker-registry.adoc#[Accessing the OpenShift Docker Registry], 
+you might encounter with following authorization error when trying to push an image to exposed registry.
+
+----
+$ docker login -u developer -p `oc whoami -t` 172.30.1.1:5000
+Login Succeeded
+$ docker pull busybox
+$ docker tag busybox 172.30.1.1:5000/myproject/busybox:latest
+$ docker push 172.30.1.1:5000/myproject/busybox:latest
+The push refers to a repository [172.30.1.1:5000/myproject/busybox]
+6a749002dd6a: Preparing 
+unauthorized: authentication required
+----
+
+Reason: An incompatibility between the docker client shipped with Fedora, CentOS or RHEL and Docker daemon which is running inside the {project} VM.
```

Comment:
Docker needs to be capitalized for "Docker client". Also use "the" before "Docker daemon".